doc/vm-tools/qvm-copy.rst

```diff
@@ -1,6 +1,25 @@
+==============
+qvm-copy
+==============
+
+NAME
+====
+qvm-copy - copy specified files to specified destination VM
```

Comment:
Not "specified" really, the user will be prompted interactively for the destination.